src/request/src/cache.js

```diff
@@ -70,9 +68,9 @@ export default class CacheRequest extends Request {
 
   set url(urlString) {
     super.url = urlString;
-    let pathname = global.decodeURIComponent(url.parse(urlString).pathname);
-    const urlParts = pathname.replace(/^\//g, '').split('/');
-    // "pathname" has the following form "/namespace/appKey/collection/id"
+    const pathname = global.decodeURIComponent(url.parse(urlString).pathname);
+    const urlParts = pathname.replace(/^\//, '').split('/');
```

Comment:
Since the regexp is anchored with the symbol for start of line, it cannot occur anywhere else in the string and doesn't need to be global, I think. Should I change it back?